src/constants/constants.js

```diff
@@ -10,6 +10,13 @@ export const UINT256_UINT32_DIVISOR_BN = UINT256_MAX_BN.div(UINT32_MAX_BN);
 
 export const SCHEMA_CONTEXT = 'http://schema.org/';
 
+export const PRIVATE_ASSERTION_PREDICATE =
+    'https://ontology.origintrail.io/dkg/1.0#privateAssertionID';
+
+export const COMMIT_BLOCK_DURATION_IN_BLOCKS = 5;
+export const BLOCK_TIME = 12;
```

Comment:
Bad thing

Reply:
Why, please add more context and reasons when leaving comments